config/runtime.exs

```diff
@@ -111,6 +124,13 @@ config :lightning,
        System.get_env("INIT_PROJECT_FOR_NEW_USER", "false")
        |> String.to_atom()
 
+# If you've booted up with a SENTRY_DSN environment variable, use Sentry!
+config :sentry,
+  filter: Lightning.SentryEventFilter,
+  environment_name: config_env(),
+  included_environments:
+    if(System.get_env("SENTRY_DSN"), do: [config_env()], else: [])
+
```

Comment:
Why is this here? Sentry is/was configured at the bottom of the file?

Reply:
Odd locally it still is at the bottom, I don't have a clue why this is here.

Reply:
This is an error,  didn't notice it before